src/vm/gdbjit.cpp

```diff
@@ -106,13 +107,15 @@ GetTypeInfoFromTypeHandle(TypeHandle typeHandle, NotifyGdb::PTK_TypeInfoMap pTyp
                 else
                 {
                     PTR_BYTE base = 0;
-                    if (!pField->IsRVA())
+                    MethodTable* pMT = pField->GetEnclosingMethodTable();
+                    base = pField->GetBaseInDomainLocalModule(pMT->GetDomainLocalModule(NULL));
```

Comment:
I tried to use this, but `GetBase()` doesn't exist if `DACCESS_COMPILE` is defined